apps/sms/views/new_message/style/recipients.css

```diff
@@ -97,7 +97,7 @@ Do not remove.
   width: auto;
 }
 
-.recipient[contenteditable].email {
+.recipient[contenteditable=false].email {
```

Comment:
Yep, still can reproduce it - enter email recipient, assimilate, press backspace to edit it